main/background.ts

```diff
@@ -46,13 +46,17 @@ function metric_string_parse(item): number | null {
   if (!item) return null;
   return +item.metrics[0].value;
 }
-ipcMain.on("metrics", async (event) => {
+
+// better dev quality, temp solution
+async function getMetrics(): Promise<Metrics> {
+  console.log("DEBUG: getMetrics start");
```

Comment:
we should consider using logger library (like winston or pino) in order to include debug logs but turn them down in releases